src/app/shared/components/search/search.component.scss

```diff
@@ -0,0 +1,26 @@
+.search {
+  display: inline-block;
+  position: relative;
+
+  ::ng-deep .mat-form-field-appearance-outline .mat-form-field-wrapper {
```

Comment:
try not to use ::ng-deep - it's deprecated and should be avoided. 
we can use it only if there is no other way to achieve desired result except ::ng-deep